### PR TITLE
[feat] Add Acme integration

### PR DESCRIPTION
**Pull request by tmynn**
Added Acme integration

### CODE REVIEW

**Review by alberttorosyan — COMMENTED** [review]
@tmynn, changes look good! left a couple of comments

**Comment by alberttorosyan** [comment]
@tmynn please also fix the flake8 errors:
Screen Shot 2023-01-04 at 14 01 30

File: aim/sdk/adapters/acme.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                    system_tracking_interval=self.system_tracking_interval,
+                    log_system_params=self.log_system_params,
+                )
+                self._run_hash = self._run.hash
+
+        if args:
+            for key, value in args.items():
+                self._run.set(key, value, strict=False)
+
+    def track(self, logs, context, step=None):
```
Comment (mihran113) [inline aim/sdk/adapters/acme.py:53]: For custom values tracking via maybe it's better to not force passing `context`?

File: docs/source/quick_start/integrations.md [hunk 2]
```diff
@@ -356,6 +356,45 @@ Check out a simple objective optimization example [here](https://github.com/aimh
...
+Aim provides a built in callback to easily track [Acme](https://dm-acme.readthedocs.io/en/latest/) trainings.
+It takes few simple steps to integrate Aim into your training script.
+
+Step 1: Explicitly import the `AimCallback` and `AimWritter` for tracking training metadata.
+
+```python
+from aim.sdk.acme import AimCallback, AimWritter
+```
+
+Step 2: Initialize a Aim Run via `AimCallback`, and create a logg factory using the Run.
```
Comment (mihran113) [inline docs/source/quick_start/integrations.md:370]: Some typos here:
```
a Aim Run -> an Aim Run,
logg -> log
```

File: aim/sdk/adapters/acme.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+        self._run.track(logs, step=step, context=context)
+
+    def close(self):
+        if self._run and self._run.active:
+            self._run.close()
+
+
+class AimWritter(Logger):
```
Comment (mihran113) [inline aim/sdk/adapters/acme.py:61]: I guess this should have been an `AimWriter`, right? 
Please also make sure to fix the mentions in the docs and examples as well.

**Review by mihran113 — CHANGES_REQUESTED** [review]

**Review by mihran113 — APPROVED** [review]

**Review by alberttorosyan — APPROVED** [review]
lgtm

**Pull request merged**